network-api/networkapi/wagtailpages/templates/wagtailpages/blog_index_page.html

```diff
@@ -7,52 +7,82 @@
-            {% for block in page.callout_box %}
-              {% include_block block %}
-            {% endfor %}
-          </div>
-        {% else %}
-          <div class="tw-col-span-12 large:tw-col-span-4">
-            {% include "./fragments/blog_card.html" with page=feature.blog %}
+    <div class="tw-grid tw-grid-cols-12 tw-gap-5 tw-mb-1 medium:tw-mb-7">
+      {% with divider_classes="tw-border-b tw-pb-5 tw-border-gray-20 medium:tw-border-b-0" %}
+       
+        {% for feature in page.specific.featured_pages.all %}
+          {% if forloop.first %}
+            <div class="tw-col-span-12 {{divider_classes}}">
+              {% include "./fragments/blog_index_feature.html" with blog_page=feature.blog %}
+            </div>
+          {% elif forloop.counter == 5 %}
+            <div class="tw-col-span-12 large:tw-col-span-7 large:tw-pb-7">
+              {% include "./fragments/blog_card.html" with page=feature.blog %}
+            </div>
+            <div class="tw-col-span-12 large:tw-col-span-5 xlarge:tw-pl-3">
+              {% for block in page.callout_box %}
+                {% include_block block %}
+              {% endfor %}
+            </div>
+          {% else %}
+            <div class="tw-col-span-12 large:tw-col-span-4 {{divider_classes}}">
+              {% include "./fragments/blog_card.html" with page=feature.blog %}
+            </div>
+          {% endif %}
+        {% endfor %}
+
+        {% for video_post in page.specific.featured_video_post.all %}
+          <div class="tw-col-span-12 tw-pt-5 {{divider_classes}}">
+            {% include "./fragments/blog_index_feature.html" with blog_page=video_post.blog_page video_url=video_post.video_url %}
           </div>
-        {% endif %}
-      {% endfor %}
-
-      {%  for video_post in page.specific.featured_video_post.all %}
-        <div class="tw-col-span-12">
-          {% include "./fragments/blog_index_feature.html" with blog_page=video_post.blog_page video_url=video_post.video_url %}
-        </div>
-      {% endfor %}
-    </div>
+        {% endfor %}
 
+      {% endwith %}
+    </div>
   {% endif %}
 {% endblock %}
 
 {% block subcontent %}
   {% if not filtered %}
-    <h2 class="tw-uppercase tw-text-3xl tw-font-bold tw-font-sans tw-text-gray-80 tw-mb-5">{% trans "Latest Blog Posts" %}</h2>
-    <div class="row">
-      {% include "./fragments/entry_cards_with_related_topics_callout.html" %}
-    </div>
-  {% else %}
-    <div class="row">
-      {% include "./fragments/entry_cards.html" %}
-    </div>
+    <h2 class="tw-hidden tw-uppercase tw-text-3xl tw-mb-0 tw-font-bold tw-font-sans tw-text-gray-80 medium:tw-block">{% trans "Latest Blog Posts" %}</h2>
   {% endif %}
+
+  <div class="row {% if filtered %} -tw-mt-5 {% endif %} ">
+    <ul id="index-entries" class="tw-grid medium:tw-grid-cols-2 large:tw-grid-cols-3 tw-gap-x-5 tw-gap-y-4 tw-list-none tw-divide-y tw-divide-gray-20 tw-px-[15px] medium:tw-divide-y-0">
```

Comment:
On this loop however, I am using the tailwind divide class, as this section gives the user an option to append more cards to the list. To remove the border from the top and bottom of the "related topics cutout" I am using the `tw-border-none` class which removes the top divider, then using `tw-pb-7 -tw-mb-6 tw-bg-white` to position the div over the bottom divider, making it dissapear. 

I will admit it feels a little fragile but I am open to suggestions :D